src/lumigo_opentelemetry/libs/sampling.py

```diff
@@ -72,107 +96,120 @@ def get_description(self) -> str:
 )
 
 
-def _extract_url(attributes: Attributes) -> Optional[str]:
-    if attributes is None:
-        return None
+def does_endpoint_match_client_filtering_regexes(endpoint: str) -> bool:
```

Comment:
This and `does_endpoint_match_server_filtering_regexes` are almost identical, with the difference being the env-var. I'd consider consolidating them to a single method and just passing the env-var name as a parameter

Reply:
Correct, I consolidated them